Packs/Akamai_WAF/ReleaseNotes/2_1_0.md

```diff
@@ -0,0 +1,17 @@
+
+#### Integrations
+
+##### Akamai WAF
+
+- %%UPDATE_RN%%
+- Updated the Docker image to: *demisto/akamai:1.0.0.50552*.
+- Updated the description for function "list_papi_property_bygroup".
+- Updated the function "list_papi_property_bygroup_ec" to return more context values.
+- Updated the command "get_appsec_config_latest_version_command" to return an error if the security configuration with the name provided is not found.
+- Added a new command "list_papi_property_bygroup_command": Lists properties available for the current contract and group.
+- Added a new command "get_papi_property_byname_command": Get papi property info without default rule. To get default rule, please use "get-papi-property-rule" command.
```

Comment:
Delete the lines below since I added them as second level bullets in the previous comment.  It's just easier to read that way.

Reply:
Removed as per suggestion.